manpages/lumina-mediaplayer.8

```diff
@@ -1,43 +1,60 @@
-.Dd October 27, 2017
+.Dd October 31, 2017
 .Dt LUMINA-MEDIAPLAYER 8
 .Os 1.3.3
 
 .Sh NAME
 .Nm lumina-mediaplayer
-.Nd is a graphical utility to play media files and stream online radio services.
+.Nd a graphical utility to play media files and stream online radio
+services.
 
 .Sh SYNOPSIS
 .Nm
-.Op <FileName1 FileName2  ... >
-.Nm
-.Op <argument>
+.Op Fl -pandora
+.Op Ar FILES
+
 
 .Sh DESCRIPTION
 .Nm
-opens up the graphical interface for the media player. The user can sync a Pandora account
-and stream music. The user can also create and manage a playlist of media files from their system.
+opens up the graphical interface for the media player.
+The user can sync a Pandora account and stream music.
+The user can also create and manage a playlist of media files from their
+system.
 .Pp
-The command can take multiple file names at a time and add them into a playlist. It can
-open .mp3, .mp4, .flac, .acc, .mpeg2, and mpeg4 files.
-Other than using the gui, the user is able to add a song by specifying a file name
-or multiple file names.
+The command can take multiple file names at a time and add them into a
+playlist.
+It can open .mp3, .mp4, .flac, .acc, .mpeg2, and mpeg4 files.
+Other than using the gui, the user is able to add a song by specifying
+a file name or multiple file names.
```

Comment:
Let's rework the sentence to avoid the comma: The user can use the command line to add songs by listing media file names.